armi/reactor/composites.py

```diff
@@ -1392,6 +1392,26 @@ def getAncestor(self, fn):
         else:
             return self.parent.getAncestor(fn)
 
+    def getAncestorWithDistance(
```

Comment:
Maybe `getAncestorAndDistance`? With distance seems to imply to me that you're specifying a distance .

Reply:
Good call